### ISSUE TITLE
Get Effects dialog: accessibility: insufficient contrast for text on some buttons.

### ISSUE THREAD

**Issue by DavidBailes** [audacity/audacity#8247]
### Steps to reproduce

Using a contrast checker, eg the color contrast analyzer in the accessibility insights for windows app, check the contrast of the text in the "get it on musehub", and "become a partner" buttons.

### Expected behavior

The contrast is at least 4.5:1, so as to pass the following criterion:
https://www.w3.org/WAI/WCAG22/Understanding/contrast-minimum

### Actual behavior

The contrast is 3.95:1

### Audacity Version

Audacity 3.7.2

### Operating system

Windows
